src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Core/Services/AuthorizedPartiesServiceEf.cs

```diff
@@ -211,17 +209,44 @@ public async Task<IEnumerable<Guid>> GetPartyFilterUuids(IEnumerable<BaseAttribu
 
                     // Directly adds the uuid we don't bother checking existence here
                     partyUuids.Add(partyUuid);
+
+                    break;
+                case AltinnXacmlConstants.MatchAttributeIdentifiers.PartyUuidAttribute:
+                case AltinnXacmlConstants.MatchAttributeIdentifiers.OrganizationUuid:
+                    if (!Guid.TryParse(partyAttribute.Value, out partyUuid))
+                    {
+                        throw new ArgumentException(message: $"Not a well-formed uuid: {partyAttribute.Value}", paramName: nameof(partyAttributes));
+                    }
+
+                    var uuidEntity = await repoService.GetEntity(partyUuid, cancellationToken);
+                    if (uuidEntity != null)
+                    {
+                        partyUuids.Add(uuidEntity.Id);
+
+                        if (uuidEntity.ParentId.HasValue)
+                        {
+                            // Also add parent uuid to cover subunit filters
+                            partyUuids.Add(uuidEntity.ParentId.Value);
```

Comment:
Mulig jeg ikke helt er med her, men burde det ikke vært child id. Hvis problemet er at man har delegert fra hovedenhet og man mangler tilsvarende rettighet for barnet?

Reply:
Tror jeg forstår. Dette grunnet dialogporten og måte de autoriserer.

Reply:
Problemet oppstår når delegering er gjort fra hovedenhet ja, men filteret er for underenhet.

Da må vi også inkludere hovedenhet/parent i filteret for at oppslaget av delegerte ressurser skal finne delegeringer fra begge.